Resources/views/CRUD/edit_orm_many_to_many.html.twig

```diff
@@ -9,7 +9,8 @@ file that was distributed with this source code.
 
 #}
 {% if sonata_admin.field_description.hasassociationadmin %}
-    <div id="field_container_{{ id }}" class="field-container {{ form.vars["attr"]["class"] is defined ? form.vars["attr"]["class"] }}">
+    <div id="field_container_{{ id }}"
+		class="field-container {{ form.vars["attr"]["class"] is defined ? form.vars["attr"]["class"] }}">
```

Comment:
I typically indent attributes on a new line by "1 tab" - aligning it to id would mean that we are aligning it by the length of the container tag. So it would be different for "div" and different for say "textarea" ... I think the issue is that the rest of the file is indented by spaces, while that line is indented by tabs.